keep/providers/nagios_provider/nagios_provider.py

```diff
@@ -0,0 +1,55 @@
+import logging
```

Comment:
1. you should add ALERTS_MOCK.py to this directory showing the payload of nagios alerts so other could work with it too
2. add a simple readme on how to reproduce (can I spin up a nagios locally?)